python/analysis.py

```diff
@@ -13,19 +15,24 @@
 
     parser.add_argument(
         '--day_one',
-        help="Use only first inpatient day's worth of features (DAN only)",
+        help=
+        "Also include models that only evaluated first inpatient day's worth of features",
         dest='day_one',
         action='store_true')
-    parser.add_argument('--all_days',
-                        help="Use all features in lookback period (DAN only)",
-                        dest='day_one',
-                        action='store_false')
+    parser.add_argument(
+        '--all_days',
+        help=
+        "Only compute CIs for models that used all features in lookback period",
+        dest='day_one',
+        action='store_false')
     parser.set_defaults(day_one=True)
     parser.add_argument("--outcome",
```

Comment:
@scotthlee I worked in some of your changes to the analysis script.

Basically it should do the same thing you wanted by looping over each outcome, but it also allows you to specify one or more outcomes you want specifically through a command line arg.